pkg/configReader/configReader.go

```diff
@@ -57,7 +57,7 @@ func ReadEnvVarsIntoStruct(s any) (any, error) {
 				if intValue == 0 && envValue != "" {
 					intV, err := strconv.Atoi(envValue)
 					if err != nil {
-						log.Error().Msgf("unable to convert %s to int: %s", envValue, err.Error())
+						log.Error().Err(err).Msgf("unable to convert %s to int", envValue)
```

Comment:
maybe move non-static part to end
```suggestion
						log.Error().Err(err).Msgf("unable to convert environment variable to int: %s", envValue)
```